easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -18,13 +17,22 @@ patches = ['ncurses-%(version)s_gcc-5.patch']
 
 configopts = [
     # default build
-    '--with-shared --enable-overwrite --with-termlib',
+    '--with-shared --enable-overwrite',
     # the UTF-8 enabled version (ncursesw)
-    '--with-shared --enable-overwrite --with-termlib ' +
-    '--enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/'
+    '--with-shared --enable-overwrite --enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/'
 ]
 
-libs = ["form", "menu", "ncurses", "panel", "tinfo"]
+# need to take care of $CFLAGS ourselves with dummy toolchain
+# we need to add -fPIC, but should also include -O* option to avoid compiling with -O0 (default for GCC)
+buildopts = ['CFLAGS="-O2 -fPIC"', 'CFLAGS="-O2 -fPIC"']
```

Comment:
you don't need to make this a list, just a string will work (it will be used once for every entry in `configopts`)

```python
buildopts = 'CFLAGS="-O2 -fPIC"
```